cobalt/build/configs/common.gn

```diff
@@ -53,7 +53,6 @@ v8_use_external_startup_data = false
 enable_hls_demuxer = false
 
 # Many of these come from media_options.gni.
-enable_dav1d_decoder = false
```

Comment:
Two things: Just removing it here would rely on it set to try elsewhere,
could we leave instead either an explicit set to `true` with a comment
on why SW AV1 decoding is needed, or an assertion:

`assert(enable_dav1d_decoder, "Dav1d must be enabled for AV1 decoding e.g. for Splash screen");`


Other question is: with this change, `enable_dav1d_decoder` will be set
to true by `//media`, but the SW AV1 decoding bit would be driven within
SbPlayer implementation, i.e. under `//starboard`, so I guess we should
add the `enable_dav1d_decoder = true` or the assertion to 
`starboard.gni`?